src/obstacle_avoidance/include/MyNode.h

```diff
@@ -0,0 +1,61 @@
+/*
```

Comment:
We should rename this file to something like `ObstacleAvoidance.h`, since this is no longer the sample node.